### ISSUE TITLE
Test the new mobile app synchronization process

### ISSUE THREAD

**Issue by MateStrysewske** [hzi-braunschweig/SORMAS-Project#7604]
### Problem Description

The new app synchronization process that will (probably) be introduced with 1.67 needs to be tested on a system with a lot of data. 

Prerequisite: #7303 and #7521 have been merged into development and deployed to the test instance.

### Proposed Solution

The testing process should look like this:

- Make sure that there are two surveillance officers in the same district which contains a lot of data on the test system #7266 
- Set up your testing device to use the full bandwidth available; log in with user A and verify that the initial synchronisation is completed successfully (i.e. there is no timeout, no error, and after logging in the data is shown in the lists). Document how long the sync took and how many cases and contacts where synced. Note: The initial sync also includes infrastructure. We could split this by logging in with another user first, but that's not crucial at this point.
- Log out from the app and set your testing device to limit bandwidth to Edge
- Log in with user B, let the synchronization run for 10 minutes and verify that the synchronisation is not cancelled by the app before (i.e. there is no timeout); cancel the synchronization after 10 minutes (because it would probably take a lot of time). Document how many cases/contacts where synced. Note: When just switching the user, the infrastructure data will stay on the device.
- Do another slow bandwidth sync for 5 minutes and document how many cases/contacts where synced

### Additional Information

It would be beneficial to create a concept for how this testing can be repeated before every new release, either manually or automatically. Also include learnings & suggestions on how to do this more efficient.

**Comment by bernardsilenou** [comment]
- Since the described test vary the bandwidth, it may be good to also repeat it by varying the number of records.
-  Ie generate like 500 records, do it and increase by 1000 and then by 10000

**Comment by StefanKock** [comment]
Testing will be done based on the implementation tickets, this ticket is discarded.
